File: iofile/python/lib/iofilex/filex.py


def read(filename):
	f = open(filename)
	content = f.read()

	f.close()
	return content

def lines(filename):
	f = open(filename)
	lines = ''
	for line in f:
		lines += line
	f.close()


	return lines


def write(filename, content):
	f = open(filename, 'w')
	f.write(content)
	f.close()

def writeLines(filename, lines):
	f = open(filename, 'w')
	for line in lines:
		f.write(line + '\n')

	f.close()




	

File: class-exception/java/lib/myiofile/Filex.java

package myiofile;

import java.util.*;
import java.io.*;
import java.nio.file.*;
import java.nio.charset.*;

public class Filex{

	public static String read(String filename){

		String res = null;

		try{
			byte[] bytes = Files.readAllBytes(Paths.get(filename));
			res = new String(bytes);
		}
		catch(Exception ex){
			ex.printStackTrace();
		}


		return res;

	}

	public static List<String> lines(String filename){

		List<String> res = null;

		try{
			res = Files.readAllLines(Paths.get(filename));
		}
		catch(Exception ex){
			ex.printStackTrace();
		}


		return res;

	}

	public static void write(String filename, String content){

		try{
			Files.write(Paths.get(filename), content.getBytes());
		}
		catch(Exception ex){
			ex.printStackTrace();
		}

	}

	public static void writeLines(String filename, List<String> lines){

		try{

			Files.write(Paths.get(filename), lines, Charset.forName("UTF-8"));
		}
		catch(Exception ex){
			ex.printStackTrace();
		}

	}

}


